docs/content.go

```diff
@@ -742,10 +742,59 @@ Enterprise Performance Computing (EPC)`
 	InspectUse   string = `inspect [inspect options...] <image path>`
 	InspectShort string = `Show metadata for an image`
 	InspectLong  string = `
-  Inspect will show you labels, environment variables, and scripts associated 
-  with the image determined by the flags you pass.`
+  Inspect will show you labels, environment variables, apps and scripts associated 
+  with the image determined by the flags you pass. By default, they will be shown in 
+  plain text. If you would like to list them in json format, you should use the --json flag.
+  `
 	InspectExample string = `
-  $ singularity inspect ubuntu.sif`
+  $ singularity inspect ubuntu.sif
+  
+  If you want to list the applications (apps) installed in a container (located at
+  /scif/apps) you should run inspect command with --list-apps <container-image> flag.
+  ( See https://sci-f.github.io for more information on SCIF apps)
+
+  The following environment variables are available to you when called 
+  from the shell inside the container. The top variables are relevant 
+  to the active app (--app <app>) and the bottom available for all 
+  apps regardless of the active app. Both sets of variables are also available during development (at build time).
+
+  ACTIVE APP ENVIRONMENT:
+      SCIF_APPNAME       the name for the active application
+      SCIF_APPROOT       the installation folder for the application created at /scif/apps/<app>
+      SCIF_APPMETA       the application metadata folder
+      SCIF_APPDATA       the data folder created for the application at /scif/data/<app>
+        SCIF_APPINPUT    expected input folder within data base folder
+        SCIF_APPOUTPUT   the output data folder within data base folder
```

Comment:
Why are these indented more then the other lines? Is that intentional?

Reply:
I would guess that it's reflective of the actual files - both `SCIF_APPINPUT` and `SCIF_APPOUTPUT` are subfolders of `SCIF_APPDATA`. I'm not sure how the above will render, but if the spacing is honored, it might make sense.